meshio/abaqus/_abaqus.py

```diff
@@ -92,8 +92,39 @@
 meshio_to_abaqus_type = {v: k for k, v in abaqus_to_meshio_type.items()}
 
 
+def bulk_w_includes(inp_path, bulk_str):
+    import os
+    import re
+
+    re_in = re.compile(
+        r"\*Include,\s*input=(.*?)$", re.IGNORECASE | re.MULTILINE | re.DOTALL
+    )
+    bulk_repl = dict()
+
+    for m in re_in.finditer(bulk_str):
+        search_key = m.group(0)
+        with open(os.path.join(os.path.dirname(inp_path), m.group(1)), "r") as d:
+            bulk_repl[search_key] = d.read()
+
+    for key, val in bulk_repl.items():
+        bulk_str = bulk_str.replace(key, val)
+    return bulk_str
+
+
 def read(filename):
     """Reads a Abaqus inp file."""
+
+    out = None
+    with open_file(filename, "r") as f:
+        bulk_str = f.read()
```

Comment:
This is a bit much. You read the entire file, search if for `*include`, and if found, you do something else, right?

Reply:
That's correct. I am definitively open for suggestions! Maybe, "bulk_w_include" should be kicked off only IF you encounter a line containing *include flag when using the existing readbuffer? And instead of reading the entire file in memory you do a new read_buffer on the referenced file and appending it contents with the existing read_buffer stuff? I think that would remove any need to read the entire file in memory.

Reply:
Sounds better. The question is where in an abaqus file you can `include` things. Could there be an abaqus file that lists some data explicitly, and other data is included from other files?

Reply:
I think *include can be used everywhere (also in included files). So perhaps some use of recursive function would be helpful? I can take a stab at it and see what I come up with?

Reply:
Yeah, it should be recursive somehow then. Sounds like a bigger change, do you want to work on it and submit another PR?

Reply:
I hacked away and made something that worked with sample files I tested it on locally, but I see that it fails on the internal checks. I might have been a bit premature when pushing that last commit. I will give it a bit more time, and try to come up with a better solution. If you want, feel free to close this PR and I'll just create a new one once I have something a bit more robust. Ideally I do not want to touch the original implementation.

Reply:
Alright, let's close this then. Open a new PR when ready.